tests/toplevel-tests.lisp

```diff
@@ -0,0 +1,66 @@
+(in-package #:coalton-tests)
+
+(deftest test-parse-package ()
```

Comment:
I think before this goes through at least one example in `small-coalton-programs` should be retrofitted with this package form, or maybe `thih` since it's already incorporated into CI.

Reply:
added a version of thih.coal and will feed it to the parser in a test case (upcoming)

Reply:
actually this is now 'fact-fib.coal'